### PR TITLE
Fixed URL Issues  

### PR DESCRIPTION
**Pull request by chanchalpatil**
Fixed issues with URLs

### CODE REVIEW

File: contents/events/hacktoberfest-2020/index.mdx [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
-
-Details of the event:
+- Get your first pull request merged on GitHub!
+- Better of Understanding the importance of Git and GitHub in the industry.
+- Connecting to organizations, reading their code and contributing gives an idea of industry standard code.
+- Great opportunity to work with developers who are a part of local communities and work towards a shared goal.
 
-Host: Open Source Lab vvce
+We provide peer to peer mentoring and guide you through your journey in Open Source and techs of all kinds!! \
+Join our <a href="https://discord.gg/rRfUpga" >Discord Channel</a> for event updates! \
```
Comment (sanjithpk) [inline contents/events/hacktoberfest-2020/index.mdx:37]: Is the \ a typo?

**Comment by chanchalpatil** [comment]
it was for line break.
Nevermind, remove it.

File: contents/events/hacktoberfest-2020/index.mdx (hunk 1 above)
Comment (sanjithpk) [reply contents/events/hacktoberfest-2020/index.mdx:37]: yeah you can remove it at the last,
image
BTW you can check the preview here

Comment (chanchalpatil) [reply contents/events/hacktoberfest-2020/index.mdx:37]: ha, checked it and edited as well then. Now, I don't know whether the checks are fulfilled. Some glitches here

Comment (chanchalpatil) [reply contents/events/hacktoberfest-2020/index.mdx:37]: Oh , All checks passed now

Comment (chanchalpatil) [reply contents/events/hacktoberfest-2020/index.mdx:38]: Any more changes @sanjithpk ?

Comment (sanjithpk) [reply contents/events/hacktoberfest-2020/index.mdx]: did you fix it? I can't see that commit

Comment (chanchalpatil) [reply contents/events/hacktoberfest-2020/index.mdx]: Ah yes, done now. It showed properly in preview. But did not work at the website

**Comment by sanjithpk** [comment]
All checks have passed, good job.

**Comment by chanchalpatil** [comment]
Thank you :thumbsup:

**Pull request merged**